cdap-app-fabric/src/main/java/co/cask/cdap/internal/app/runtime/schedule/constraint/LastRunConstraint.java

```diff
@@ -30,19 +30,19 @@
  */
 public class LastRunConstraint extends ProtoConstraint.LastRunConstraint implements CheckableConstraint {
 
-  public LastRunConstraint(long millisSinceLastRun) {
-    super(millisSinceLastRun);
+  public LastRunConstraint(long secsSinceLastRun) {
```

Comment:
It can be seconds, sure. But I think whatever we choose for LastRunConstraint, we should choose the same for DelayConstraint.